src/templates/video.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Video as IVideo } from '~src/interfaces/video';
+
+type VideoProps = {
+    pageContext: {
+        data: IVideo;
+    };
+};
+
+const Video = ({ pageContext: { data: video } }: VideoProps) => {
+    console.log(video);
+
+    return <>666 TEST</>;
```

Comment:
We should just return null or something else instead here for now.

Reply:
Yeah, but for final state of PR this line will disappear. I will replace this by component form another PR.